2. Services/2.2 Processings/2.2 Processings.md

```diff
@@ -21,7 +21,7 @@ TryCatch(async () =>
         retrievedStudent.Id == student.Id);
 
     return studentExists switch {
-        false => await this.studentService.RegisterStudentAsync(student),
+        false => await this.studentService.AddStudentAsync(student),
```

Comment:
This should stay as `RegisterStudentAsync` the idea here is that the language is more business specific than technical at the `Processing` layer.

Reply:
Fixed